packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -537,22 +488,6 @@ class AppDomain extends Domain {
     });
   }
 
-  Future<List<Map<String, dynamic>>> discover(Map<String, dynamic> args) async {
-    final String deviceId = _getStringArg(args, 'deviceId', required: true);
-
-    final Device device = await daemon.deviceDomain._getDevice(deviceId);
-    if (device == null)
-      throw "device '$deviceId' not found";
-
-    final List<DiscoveredApp> apps = await device.discoverApps();
```

Comment:
I'm curious if `device.discoverApps()` is called from anywhere now? If no, we should add a todo: to the declaration.

We'll need similar but different functionality for add2app.

Reply:
Ok, I've added a TODO and a link here. I'll land this with that TODO and come back to it depending on the response from @jason-simmons.